webpack.config.babel.js

```diff
@@ -8,7 +8,7 @@ const env = process.env.NODE_ENV || 'development'
  * Base Configuration.
  */
 const config = {
-  devtool: 'cheap-eval-source-map',
+  devtool: env === 'production' ? '#eval' : '#eval-source-map',
```

Comment:
hey @JFusco,

was more asking if we should add here instead of the base config? https://github.com/makeshift-array/build-tools/blob/a4d2e93033ed3e6af833e5cb54012712c3dedf36/webpack.config.babel.js#L83